index.html

```diff
@@ -14,6 +14,373 @@
   <title>Itumeleng Rantsane | Portfolio </title>
 </head>
 
+<<<<<<< HEAD
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/203607059-4d4a6cf2-0f00-4d9e-80f7-48ef0edd5242.PNG)

- [ ] Kindly remove that **<<<<<< Head** shown in the image above ⬆️to make your project look professional as suggested by the previous reviewer 😊